pkgs/development/tools/misc/devspace/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "devspace";
+  version = "5.18.4";
+
+  src = fetchFromGitHub {
+    owner = "loft-sh";
+    repo = "devspace";
+    rev = "v${version}";
+    sha256 = "0s5117cgxgrxfki5drvg6d22dvrjffa03bi644zdl1p631r599r1";
+  };
+
+  vendorSha256 = null;
+
+  doCheck = false;
+
```

Comment:
some `ldflags` are needed so that `devspace --version` works: https://github.com/loft-sh/devspace/blob/61e928dc3b90159a452912e0d987a3ebb0a21ce6/hack/build-all.bash#L32

Reply:
Probably a dumb question, but I don't think it's worth to add the timestamp and the commit hash, since the `--version` command can work without them and don't display them. In fact, I think we can just set the version ldflag

Reply:
Sounds right

Reply:
Maybe we still want to set -s -w ? I guess it can help e.g. in reducing binary size.

Reply:
Done :heavy_check_mark: